src/lib/Libpython/pbs_python_external.c

```diff
@@ -172,7 +172,7 @@ pbs_python_ext_start_interpreter(
 
 	Py_NoSiteFlag = 1;
 	Py_FrozenFlag = 1;
-	Py_OptimizeFlag = 2;            /* TODO make this a compile flag variable */
+	Py_OptimizeFlag = 0;            /* TODO make this a compile flag variable */
```

Comment:
Having optimization sets removes doc strings, all the asserts and __debug__ codes. Let me see if I can remove it.

From doc: A program doesn’t run any faster when it is read from a .pyc file than when it is read from a .py file; the only thing that’s faster about .pyc files is the speed with which they are loaded.

Refer section: 6.1.3. “Compiled” Python files here https://docs.python.org/3/tutorial/modules.html